src/components/settings-table/dataset-table-row.js

```diff
@@ -19,7 +21,12 @@ const DataSetTableRow = ({
                 scope="row"
                 className={dataTableStyles.dataTable__rows__row__column}
             >
-                {dataRow.option} ({periodType})
+                <p>
+                    {dataRow.option} ({periodType})
+                </p>
+                <em className={tableTitleStyles.attributeLabel}>
+                    {i18n.t('Default:')} {defaultValue}
```

Comment:
Suggest using interpolation for this translation:

`i18n.t('Default: {{defaultValue}}', {defaultValue})`

Reply:
as you suggest I'm separating this phrase in 3 parts:
`{i18n.t('Default')} : {i18n.t('{{defaultValue}}', {defaultValue})}`